src/AppBundle/Twig/SourceCodeExtension.php

```diff
@@ -95,7 +95,7 @@ private function getTemplateCode()
 
     private function getTemplateRelativePath()
     {
-        return 'app/Resources/views/'.$this->template->getTemplateName();
+        return 'app'.DIRECTORY_SEPARATOR.'Resources'.DIRECTORY_SEPARATOR.'views'.DIRECTORY_SEPARATOR.$this->template->getTemplateName();
```

Comment:
This is not needed here, given that ``Twig_Template::getTemplateName()`` does not use ``DIRECTORY_SEPARATOR`` anyway.

On a side note, this implementation is fragile: if your template uses a namespaced path (``@WebProfiler/Collector/time.html.twig`` for instance) or any other loader than the Twig_Loader_Filesystem (the ``TwigBundle:Exception:exception.txt.twig`` notation for instance, or an anonymous template created by the ``{% embed %}`` tag for instance), the template name will not be a relative path at all.